README.md

```diff
@@ -32,19 +33,12 @@ Then, load http://localhost:3000/ in your browser to access the app.
 
 ### Install and run application for production
 ```shell
+nvm use
```

Comment:
I think you can remove because this command will be in it's own shell command. Also NVM is great for local development but for production we don't know if they'll use NVM.